backend/api/Dockerfile

```diff
@@ -1,10 +1,10 @@
 # Generate client code (go & json) from API protocol buffers
-FROM golang:1.13.1-stretch as generator
+FROM golang:1.15.10 as generator
```

Comment:
When you update the dockerfile, you should push a new version of the api generator image to gcr.io/ml-pipeline-test, so that others keep using the same image as you did.

Push the image via https://github.com/kubeflow/pipelines/blob/4a4296a1fed4f6c5a3ec01fa023e61fb2fe683ec/backend/api/Makefile#L28-L32,
Update prebuilt image in https://github.com/kubeflow/pipelines/blob/4a4296a1fed4f6c5a3ec01fa023e61fb2fe683ec/backend/api/Makefile#L6

(you might also want to document these in backend/api/README.md)